dispatch-build-bottle/action.yml

```diff
@@ -0,0 +1,124 @@
+name: Dispatch build bottle
+inputs:
+  formula:
+    description: "Formula name"
+    required: true
+  issue:
+    description: "Issue number, where comment on failure would be posted"
+    required: true
+  ignore-errors:
+    description: "Ignore errors (default: false)"
+    required: false
+  macos:
+    description: "macOS Version"
+    required: false
+  name:
+    description: "Name"
+    required: false
+  sender:
+    description: "Sender"
+    required: true
+  upload:
+    description: "Whether to upload built bottles or not (default: false)"
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: ${{ inputs.formula }}
+      id: print_details
+      run: |
+        echo formula=${{ inputs.formula }}
+        echo sender=${{ inputs.sender }}
+        echo name=${{ inputs.name }}
+        echo ignore-errors=${{ inputs.ignore_errors }}
+        echo version=${{ inputs.macos }}
+        echo issue=${{ inputs.issue }}
+        echo upload=${{ inputs.upload }}
+        echo "::set-output name=repo::$(brew --repo ${{github.repository}})"
+
+    - name: Set up Homebrew
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - name: Run brew test-bot --only-cleanup-before
+      if: github.repository == 'Homebrew/homebrew-core'
+      id: test-bot-only-cleanup-before
+      run: brew test-bot --only-cleanup-before
+
+    - name: Run brew test-bot --only-setup
+      if: github.repository == 'Homebrew/homebrew-core'
+      id: test-bot-only-setup
+      run: brew test-bot --only-setup
+
+    - name: Build bottles
+      if: always()
+      env:
+        HOMEBREW_GITHUB_API_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      run: |
+        mkdir ~/bottles
+        cd ~/bottles
+        brew test-bot \
+          --only-formulae \
+          --keep-old \
+          --skip-recursive-dependents \
```

Comment:
`--skip-recursive-dependents` is not present in `homebrew-core`'s action's `step` ,
Does this step need to be separated  based on `github.repository` ?